src/main/java/com/example/SurveyController.java

```diff
@@ -1,7 +1,107 @@
 package com.example;
 
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
 
-@RestController
+import java.util.ArrayList;
+import java.util.Optional;
+
+@Controller
+@RequestMapping("/")
```

Comment:
should this be here

Reply:
At this point no, each of the controller methods have a URL pattern specified anyways. Once the testing is complete we can remove it before merging.

Reply:
ok